readme.md

```diff
@@ -13,30 +13,25 @@
 
 ## Opening Framing (15/15)
 
-Let's list out the things we've covered in class...
+Let's start out by listing the things we've covered in the first 3 units of WDI.
 
----
 
-This is SO much stuff. We've pretty much covered the entire stack. Everything we
-cover from here on out is an extension of what we've already learned or a
-different language than we're used to. Today we'll be talking about [expressJS](https://expressjs.com/)
-the "e" in the MEAN stack. Which incidentally is super buzz wordy right now.
+We've moved in and out of several different technological layers with increasing comfort over the course of the class. We've covered the entire stack almost twice over now. We're about to fill in that missing piece by talking about the Javascript back-end.
+
+Everything we cover from here on out is an extension of what we've already learned or a different language than we're used to. Today we'll be talking about [expressJS](https://expressjs.com/) (the **E** in the M**E**AN stack) and Node.js (the **N** in MEA**N**).
 
```

Comment:
Nice framing! I think you can cut out `...or a different language that we're used to` in the second graf, first sentence. It's a little confusing because JS isn't new to them and everything before that part captures the intent.